spec/serializers/search/user_serializer_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe Search::UserSerializer do
+  let(:user) { create(:user) }
+
+  it "serializes a podcast episode" do
+    data_hash = described_class.new(user).serializable_hash.dig(:data, :attributes)
+    expect(data_hash.keys).to include(:id, :name, :path, :username)
```

Comment:
This serializer spec is less about checking all the fields and more about making sure every field we ask for from a user is present and we dont raise an error trying to get it. This will raise an error for example if we remove a field from the user model without remembering to do it in the serializer as well.